code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -290,6 +290,9 @@ meteor_act
 		var/obj/item/organ/external/affecting = get_organ(zone)
 		var/hit_area = affecting.name
 
+		if (O.ignites_with_touch)
+			IgniteMob()
```

Comment:
does this proc check the fire stacks?

Reply:
IgniteMob only ignites when there are fire_stacks present.